x-pack/plugins/security/server/index.ts

```diff
@@ -18,6 +18,7 @@ export {
   AuthenticationResult,
   BasicCredentials,
   DeauthenticationResult,
+  OIDCAuthenticationFlow,
```

Comment:
__note:__ I'll remove this export as soon as we move routes to new platform security plugin, it's just for internal use.